### ISSUE TITLE
[BUG] custom SSL cloudflare 

### ISSUE THREAD

**Issue by farzadha2** [bunkerity/bunkerweb#559]
**Description**
error on docker compose also error on reading the letsencrypt 
in the original it says 
```
      - CUSTOM_HTTPS_CERT=/certs/live/example.com/fullchain.pem
      - CUSTOM_HTTPS_KEY=/certs/live/example.com/privkey.pem
```
when it should be 
```
      - CUSTOM_SSL_CERT=/certs/live/example.com/fullchain.pem
      - CUSTOM_SSL_KEY=/certs/live/example.com/privkey.pem
```

**Logs**

```
 ⚠️  - Certificate file /certs/live/domain.com/fullchain.pem is not a valid file, ignoring the custom certificate
- CUSTOM-CERT - ℹ️  - No change for certificate /certs/live/domain.com/fullchain.pem
 CUSTOM-CERT - ℹ️  - Checking certificate /certs/live/domain.com/fullchain.pem ...
 CUSTOM-CERT - ⚠️  - Certificate file /certs/live/domain.com/fullchain.pem is not a valid file, ignoring the custom certificate
 CUSTOM-CERT - ℹ️  - No change for certificate /certs/live/domain.com/fullchain.pem
```

**Comment by fl0ppy-d1sk** [comment]
Hello @farzadha2,

Documentation has been updated, thanks.

Regarding the file error, did you give permissions to the scheduler user to read it (`chown 101:101 file.pem` on the host) ?

**Comment by farzadha2** [comment]
thanks it seems to be fixed in 1.5.2
